<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DREAM3DUpdateCheckDialog</class>
 <widget class="QDialog" name="DREAM3DUpdateCheckDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check for Updates</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QComboBox" name="howOften">
       <item>
        <property name="text">
         <string>Daily</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Weekly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monthly</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="manually">
       <property name="text">
        <string>Manually</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../Resources/icons/images/Icons.qrc">:/DREAM3D.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="automatically">
       <property name="text">
        <string>Automatically</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="messageLabel">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>How would you like to check for updates?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="checkNowBtn">
         <property name="text">
          <string>Check Now</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="currentVersion">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: italic 14pt;
font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>Latest Version:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>Current Version:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="latestVersion">
       <property name="styleSheet">
        <string notr="true">QLabel {
	font: italic 14pt;
font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="feedbackText">
       <property name="text">
        <string>D</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>automatically</sender>
   <signal>toggled(bool)</signal>
   <receiver>howOften</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
